scripts/mail-alarm

```diff
@@ -480,6 +491,46 @@ class HAHostFailedETG(EmailTextGenerator):
 
         return body_pattern % self.text
 
+class SRIOThroughputTotalAlertETG(EmailTextGenerator):
+  def __init__(self, cls, obj_uuid, value, alarm_trigger_period, alarm_trigger_level, sr_uuid, mail_language):
+        super(SRIOThroughputTotalAlertETG, self).__init__(mail_language, 'sr_io_throughput_total_alert')
+        if alarm_trigger_period is None:
+            alarm_trigger_period = 60
+        if cls == 'Host':
+            self.params = get_host_params(obj_uuid)
+        else:
+            raise Exception, "programmer error"
+        self.value = value
+        self.alarm_trigger_period = alarm_trigger_period
+        self.alarm_trigger_level = alarm_trigger_level
+        self.sr_uuid = sr_uuid
```

Comment:
@mseri -- Yes I did want to use it that way. But after I checked alert message in XenCenter notification, I found there is no uuid info in such message.
I forgot to remove it from class member, so thanks for your suggestion.